src/auth/components/form/container.tsx

```diff
@@ -0,0 +1,20 @@
+import { StyledLink } from './link'
+import { PageContainer } from './PageContainer'
+
+type Props = {
+  title: string
+  children: React.ReactElement
+  link: { label: string; to: string }
```

Comment:
```suggestion

type Props = {
  title: string
  children: React.ReactElement
  link: { label: string; to: LinkProps["to"] }
```

Reply:
Nie bardzo rozumiem czym ma być LinkProps

Reply:
`import { Link, LinkProps } from 'react-router-dom';` - ale to już wrzucę taki mały commit